WordPress/Classes/ViewRelated/Post/WPStyleGuide+Posts.swift

```diff
@@ -0,0 +1,140 @@
+        progressView.trackTintColor = greyLighten20()
+        progressView.progressTintColor = mediumBlue()
+        progressView.tintColor = mediumBlue()
+    }
+
+    class func applyRestorePostLabelStyle(_ label: UILabel) {
+        configureLabelForRegularFontStyle(label)
+        label.textColor = grey()
+    }
+
+    class func applyRestorePostButtonStyle(_ button: UIButton) {
+        configureLabelForRegularFontStyle(button.titleLabel)
+        button.setTitleColor(wordPressBlue(), for: .normal)
+        button.setTitleColor(darkBlue(), for: .highlighted)
+    }
+
+    class func applyBorderStyle(_ view: UIView) {
+        view.heightAnchor.constraint(equalToConstant: 1.0 / UIScreen.main.scale).isActive = true
+        view.backgroundColor = postCardBorderColor()
+    }
+
+    class func applyActionBarButtonStyle(_ button: UIButton) {
+        button.flipInsetsForRightToLeftLayoutDirection()
+        button.setImage(button.imageView?.image?.imageWithTintColor(grey()), for: .normal)
+        button.setTitleColor(grey(), for: .normal)
+        button.setTitleColor(darkGrey(), for: .highlighted)
+        button.setTitleColor(darkGrey(), for: .selected)
+    }
+
+    class func applySelectedBackgroundView(_ selectedBackgroundView: UIView, topMargin: CGFloat) {
+        let marginMask = UIView()
+        selectedBackgroundView.addSubview(marginMask)
+        marginMask.translatesAutoresizingMaskIntoConstraints = false
+        marginMask.leadingAnchor.constraint(equalTo: selectedBackgroundView.leadingAnchor).isActive = true
+        marginMask.topAnchor.constraint(equalTo: selectedBackgroundView.topAnchor).isActive = true
+        marginMask.trailingAnchor.constraint(equalTo: selectedBackgroundView.trailingAnchor).isActive = true
+        marginMask.heightAnchor.constraint(equalToConstant: topMargin).isActive = true
+        marginMask.backgroundColor = greyLighten30()
+    }
+
+    // MARK: - Attributed String Attributes
+
+    class var postCardTitleAttributes: [NSAttributedString.Key: Any] {
+        let font = notoBoldFontForTextStyle(.headline)
+        let lineHeight = 1.4 * font.pointSize
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.minimumLineHeight = lineHeight
+        paragraphStyle.maximumLineHeight = lineHeight
+        return [.paragraphStyle: paragraphStyle, .font: font]
+    }
+
+    class var postCardSnippetAttributes: [NSAttributedString.Key: Any] {
+        let textStyle: UIFont.TextStyle = UIDevice.isPad() ? .callout : .subheadline
+        let font = notoFontForTextStyle(textStyle)
+        let lineHeight = 1.6 * font.pointSize
+        let paragraphStyle = NSMutableParagraphStyle()
+        paragraphStyle.minimumLineHeight = lineHeight
+        paragraphStyle.maximumLineHeight = lineHeight
+        return [.paragraphStyle: paragraphStyle, .font: font]
+    }
+
+    // MARK: - Nav Bar Styles
+
+    class var navigationBarButtonRect: CGRect {
```

Comment:
This var doesn't need a getter now that it's Swift 😀. Same with `spacingBetweeenNavbarButtons`.

Maybe even `postCardBorderColor()`, what do you think?

Reply:
Done!